.gitlab-ci.yml

```diff
@@ -448,6 +449,65 @@ check-polkadot-companion-status:
   script:
     - ./.maintain/gitlab/check_polkadot_companion_status.sh
 
+pr-docker-build:            &pr-docker-build
+  stage:                           build-pr
+  tags: 
+    - kubernetes-parity-build
+  variables:
+    DOCKER_HOST:                   tcp://localhost:2375
+    DOCKER_DRIVER:                 overlay2
+    PRODUCT:                       substrate
+    DOCKERFILE:                    $PRODUCT.Dockerfile
+    CONTAINER_IMAGE:               paritypr/$PRODUCT
```

Comment:
Bump

Reply:
It's meant to keep the temp CI images. Currently me and gabriel are owners of the repo.

Reply:
Please use `paritytech`, it's meant for technical-purpose images. I can add you to that registry. The credentials for `paritytech` are already used in the other jobs in this CI config.

Reply:
@TriplEight as this job is meant to be run on every pr and docker hub account tokens are not restricted to a project using the paritytech credentials would unnecessarily expose them. Is there any other ci pushing images from non-protected branches e.g. pull requests?

Reply:
My initial idea was not to produce the same entities of `parity/substrate`, but have a separate tag in that repository. 
But it was decided that this job has to run (and push images) on every commit, so it will trash our `:latest` tag. I agree to have this separate docker registry, but with appropriate security measures. I.e. an admin service account with 2FA.

> Is there any other ci pushing images from non-protected branches e.g. pull requests?

not for this repo.